script.js

```diff
@@ -1,4 +1,62 @@
+var sciSSorsPaperStone = function () {
+  var randomNumber = Math.floor(Math.random() * 3 + 1);
+  //console.log(sciSSorsPaperStone);
+  if (randomNumber == 1) {
+    return "scissor";
+  }
+  if (randomNumber == 2) {
+    return "paper";
+  }
+  if (randomNumber == 3) {
+    return "stone";
+  }
+};
+
+var sciSSor = "scissor";
+var stoNe = "stone";
+var paPer = "paper";
+var revsciSSor = "reversed scissor";
+var revstoNe = "reversed stone";
+var revpaPer = "reversed paper";
+
+var emotiCon = function (object) {
+  if (object == sciSSor || object == revsciSSor) return " ✂️";
+  if (object == paPer || object == revpaPer) return " 🗒";
+  if (object == stoNe || object == revstoNe) return " 🪨";
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var comSPS = sciSSorsPaperStone();
```

Comment:
Can also create playerSPS variable and assign input to this variable. It then becomes very clear: 
comSPS is the computer's hand
playerSPS is the player's hand